WWDC/ImageDownloadCenter.swift

```diff
@@ -199,7 +221,21 @@ final class ImageCacheProvider {
             key = sourceURL.imageCacheKey as NSString
         }
 
-        guard fileManager.fileExists(atPath: url.path), let image = NSImage(contentsOf: url) else { return nil }
+        guard
+            fileManager.fileExists(atPath: url.path),
+            let data = try? Data(contentsOf: url),
+            let source = CGImageSourceCreateWithData(data as CFData, nil),
```

Comment:
This is an attempt to pre-decode the image, I don’t think it is actually working.